src/primitives/transaction.h

```diff
@@ -304,6 +304,16 @@ class CTxOut
     std::string ToString() const;
 };
 
+// The maximum value which is valid for expiry height, used by CTransaction and CMutableTransaction
+static constexpr uint32_t TX_EXPIRY_HEIGHT_THRESHOLD = 500000000;
+
+// Overwinter version group id is a random and unique identifier assigned
+// to a tx format version or group of soft-forking tx format versions.
+// This helps nodes disambiguate between branches using the same version number
+// but where the transaction format itself is different.
+// Forks and clones of Zcash should change this value.
```

Comment:
Will move to ZIP. I was thinking of a clone at the activation height who wanted their own v3 format, rather than after the activation height. Although they could start a brand new chain using the code, in which case they should change it even if the same v3 format to help disambiguate.